packages/odyssey-storybook/src/components/odyssey-mui/Select/Select.stories.tsx

```diff
@@ -245,6 +248,25 @@ const Template: StoryObj<SelectProps> = {
 
 export const Default: StoryObj<SelectProps> = {
   ...Template,
+  play: async ({ canvasElement, step }) => {
+    await step("Select Earth from the listbox", async () => {
+      const comboBoxElement = canvasElement.querySelector(
+        '[aria-haspopup="listbox"]'
+      );
+      if (comboBoxElement) {
+        userEvent.click(comboBoxElement);
+        const listboxElement = screen.getByRole("listbox");
+        expect(listboxElement).toBeInTheDocument();
+        const listItem = listboxElement.children[0];
```

Comment:
I'm not gonna block the PR over this, but I prefer using `.at(0)` in 2023:
```suggestion
        const listItem = listboxElement.children.at(0);
```

I use `[]` strictly for strings and symbols now.

Reply:
Does HTMLCollection type have `at`? @KevinGhadyani-Okta I'm getting an error.